### ISSUE TITLE
Remove aria-autocomplete attribute from search-textbox

### ISSUE THREAD

**Issue by abaevbog** [zotero/zotero#4932]
https://forums.zotero.org/discussion/120552/quick-search-doesnt-actually-have-popups

`search-textbox` has `aria-autocomplete` set by mozilla [here](https://searchfox.org/mozilla-esr128/source/toolkit/content/widgets/search-textbox.js#136), which ends up being not appropriate for our use cases. It tells screen readers that there should be an autocomplete popup, similar to when you edit tags, that can be navigated with arrow keys. Different screen readers announce it differently. For example, JAWS says that there is a popup, and NVDA says that there is autocomplete. In any case, it needs to be removed from all instances of `search-textbox` to avoid confusion. 

Another point in the linked post is that there is no audible announcement when itemTree results are updated after typing in quickSearch. That is something that would be great to add, though currently `aria-live` regions (which is the most reasonable way to do it) do not reliably work in chrome windows.
